src/main/java/duke/task/TaskList.java

```diff
@@ -167,4 +168,25 @@ public void delete(int i) throws IndexOutOfBoundsException {
         }
         return output;
     }
+
+    /**
+     * Views task and schedule based on a specific date.
+     * @param date Date to be searched for in the form of DD/MM/YYYY
+     * @return An ArrayList of strings representing the list of tasks based on date provided
+     */
+    public ArrayList<Task> vfilter(LocalDate date) {
```

Comment:
Please rename to a more descriptive name, such that just by looking at the method name, people know what it is suppose to do